documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -160,14 +160,60 @@ include::{testDir}/example/DisplayNameGeneratorDemo.java[tags=user_guide]
   | | +-- For example, year -1 is not supported. [OK]
   | | '-- For example, year -4 is not supported. [OK]
   | '-- if it is zero() [OK]
-  '-- A year is a leap year... [OK]
```

Comment:
Maybe this ellipsis at the end of the class name wasn't such a bad idea? 🤔

Reply:
Do you mean the 3 dots?

Reply:
Yes!